### DISCUSSION TITLE
Why two notification triggering in notification package

### DISCUSSION THREAD

**Discussion by nayeshdaggula** [mantinedev/mantine#3285] [Q&A]
Hi

Am getting two notification alerts each initiation. how to make it single? any suggestions

Thank you

**Comment by rtivital** [comment]
Do you call showNotification in useEffect?

**Reply by nayeshdaggula** [reply]
no it inside the function. that function called in useEffect. 

but for normal onclick function also triggering two alerts.

**Reply by rtivital** [reply]
useEffect is called twice in react 18.
